monitoring/Readme.md

```diff
@@ -0,0 +1,16 @@
+To plot the fields one can use the python scripts: soca_plotfield.py
```

Comment:
1 - The convention is to name that file README.md
2 - `.md` stands for markdown. Make use of it!